server/v2/cometbft/abci.go

```diff
@@ -93,9 +97,8 @@ func NewConsensus[T transaction.Tx](
 		processProposalHandler: nil,
 		verifyVoteExt:          nil,
 		extendVote:             nil,
-		chainID:                chainId,
```

Comment:
we should keep  setting those

Reply:
good catch, something wrong when i roleldback a change